cmd/aries-agentd/main.go

```diff
@@ -28,7 +28,7 @@ import (
 	"net/http"
 	"os"
 
-	"github.com/go-openapi/runtime/middleware/denco"
+	"github.com/gorilla/mux"
```

Comment:
Let's remember to do the followup where a go.mod is added to the cmd/aries-agentd folder. This way the framework's dependencies are reduced.

Reply:
updated #94